packages/scanner/package.json

```diff
@@ -56,7 +56,7 @@
   },
   "dependencies": {
     "@appland/client": "^1.3.0",
-    "@appland/models": "^1.16.1",
+    "@appland/models": "workspace:^1.18.1",
```

Comment:
Is this right?

Reply:
Yes, this will allow us to resolve to local workspace packages instead of the published `npm` package. Once it's published, it'll reference `^1.18.1`.